python/build.py

```diff
@@ -62,6 +62,10 @@ def get_python_executable():
 
 print("*** project_name = '{}'".format(project_name))
 
+try:
+    os.remove('taichi/CHANGELOG.md')
+except:
```

Comment:
instead of catch wild exceptions, how about
```suggestion
except FileNotFoundError:
```
or more conservatively, `OSError`?